contrib/inactive-windows-transparency.py

```diff
@@ -16,7 +16,11 @@ def on_window_focus(inactive_opacity, ipc, event):
     global prev_workspace
 
     focused = event.container
-    workspace = ipc.get_tree().find_focused().workspace().num
+
+    try:
+        workspace = ipc.get_tree().find_focused().workspace().num
```

Comment:
The overall change looks fine, but let's not use exceptions for control flow where an `if ...:` would suffice.

Reply:
Sounds good, tell me what you think about the new commit. Should I factorize some code or is it enough?

Reply:
I think an intermediate variable would be nice, because the `get_tree` call performs an RPC under-the-hood.